packages/expo-modules-test-core/src/getStructure.ts

```diff
@@ -250,7 +257,8 @@ function parseModuleDefinition(
       findNamedDefinitionsOfType('Prop', preparedModuleDefinition, file),
       ['view']
     ),
-    view: findAndParseView(preparedModuleDefinition, file),
+    view: findAndParseNestedClassesOfType(preparedModuleDefinition, file, 'View')[0],
```

Comment:
We'll be supporting multiple views per module at some point, so I think we can assume that already here.
```suggestion
    views: findAndParseNestedClassesOfType(preparedModuleDefinition, file, 'View'),
```

Reply:
Done with a workaround to export it as `View` instead of view name for now